ANYbotics/grid_map
# Fix Eigen Plugins compilation when linking to grid_map_core

# Purpose

Here's a proposed patch for #382. Because the original PR for adding the plugins through CMake and ament is not well documented, I really don't know how this will affect others. Please try this PR out. It will stay open for a while until I get a reasonable number of approvals it doesn't break anything. 

# Test instructions

* Check this branch out
* `colcon build --packages-up-to  issue382`


And try building your own packages that link to grid_map_core.
